index.js

```diff
@@ -37,6 +40,24 @@ module.exports = function (opts, cb) {
 		s.test = 'xo';
 	}
 
+	var cli = minimist(args);
+	var unicorn = cli.unicorn;
+
+	delete cli._;
+	delete cli.unicorn;
+	delete cli.init;
+
+	if (cli.env) {
+		cli.envs = arrify(cli.env);
+		delete cli.env;
+	}
```

Comment:
It also needs to handle `cli.global` => `cli.globals` and `cli.ignore` => `cli.ignores`.